src/app/find/locationDetails/ShelterService.jsx

```diff
@@ -0,0 +1,153 @@
+import React, { useState } from 'react';
```

Comment:
One general comment is that shelters are only one of the many service categories we have.
Streetlives also supports food providers, clothing, various types of personal care, legal services, health, mail, etc etc etc.

So, a more correct name would be maybe something like "ServiceSection".